charts/castai-agent/templates/clustervpa-configmap.yaml

```diff
@@ -21,7 +21,7 @@ data:
       "agent": {
         "requests": {
           "memory": {
-            "base": "0",
```

Comment:
@AugustinasS  you need to bump app chart versions and do the PR title according to conventions